3.10/indexing-index-basics.md

```diff
@@ -94,84 +93,81 @@ user-defined indexes.
-  created, by adding a combined index for the fields `_from` and `_to` to an edge collection.
+Vertex-centric indexes
+----------------------
 
-- **unique, sparse hash index**: all documents in the collection must have different 
-  values for the attributes covered by the unique index. Documents in which at least one
-  of the index attributes is not set or has a value of `null` are not included in the 
-  index. This type of index can be used to ensure that there are no duplicate keys in
-  the collection for documents which have the indexed attributes set. As the index will
-  exclude documents for which the indexed attributes are `null` or not set, it can be
-  used for optional attributes.
+As mentioned above, the most important indexes for graphs are the edge
+indexes, indexing the `_from` and `_to` attributes of edge collections.
+They provide very quick access to all edges originating in or arriving
+at a given vertex, which allows to quickly find all neighbors of a vertex
+in a graph.
 
-- **non-unique hash index**: all documents in the collection will be indexed. This type
-  of index is not sparse. Documents that do not contain the index attributes or that have 
-  a value of `null` in the index attribute(s) will still be indexed. Duplicate key values 
-  can occur and do not lead to unique constraint violations.
- 
-- **non-unique, sparse hash index**: only those documents will be indexed that have all
-  the indexed attributes set to a value other than `null`. It can be used for optional
-  attributes.
+In many cases one would like to run more specific queries, for example
+finding amongst the edges originating from a given vertex only those
+with a timestamp greater than or equal to some date and time. Exactly this
+is achieved with "vertex-centric indexes". In a sense these are localized
+indexes for an edge collection, which sit at every single vertex.
 
-The amortized complexity of lookup, insert, update, and removal operations in unique hash 
-indexes is O(1). 
+Technically, they are implemented in ArangoDB as indexes, which sort the 
+complete edge collection first by `_from` and then by other attributes
+for _OUTBOUND_ traversals, or first by `_to` and then by other attributes
+for _INBOUND_ traversals. For traversals in _ANY_ direction two indexes
+are needed, one with `_from` and the other with `_to` as first indexed field.
 
-Non-unique hash indexes have an amortized complexity of O(1) for insert, update, and
-removal operations. That means non-unique hash indexes can be used on attributes with 
-low cardinality. 
+If we for example have a persistent index on the attributes `_from` and 
+`timestamp` of an edge collection, we can answer the above question
+very quickly with a single range lookup in the index.
 
-If a hash index is created on an attribute that is missing in all or many of the documents,
-the behavior is as follows:
+You can create sorted persistent indexes that index the special edge attributes
```

Comment:
```suggestion
You can create sorted persistent indexes that indexes the special edge attributes
```

Reply:
"index the special edge attributes" (plural) seems correct?